client/src/Components/Common/CommentContainer/index.js

```diff
@@ -0,0 +1,69 @@
+import React, { useState, useEffect } from 'react';
+import { List } from 'antd';
+import PropTypes from 'prop-types';
+import Comment from '../Comment';
+import MainButton from '../MainButton';
+
+const commentsPerPage = 5;
+let arrayForHoldingComments = [];
+
+const CommentContainer = ({ data }) => {
+  const [commentsToShow, setCommentsToShow] = useState([]);
+  const [next, setNext] = useState(5);
+
+  const loopWithSlice = (start, end) => {
```

Comment:
since you are using functional component I encourage you to add all your variables inside the function 
```suggestion

let arrayForHoldingComments = [];

const CommentContainer = ({ data }) => {
  const [commentsToShow, setCommentsToShow] = useState([]);
  const [next, setNext] = useState(5);
const commentsPerPage = 5;
  const loopWithSlice = (start, end) => {
```

Reply:
First I want to thank you for your review. Really its beneficial.
 The delete button will appear only if the user is an admin, and for the evaluation stars ,we make it read only in the this context . The user can add his evaluation in onother place not here.